bridge_adaptivity/module/views.py

```diff
@@ -527,6 +524,17 @@ def sync_collection(request, slug):
     return redirect(reverse('module:collection-detail', kwargs={'slug': slug}) + '?back_url={}'.format(back_url))
 
 
+def update_students_grades(request, group_slug):
```

Comment:
For me more suitable use jquery for this operation

Reply:
Thanks, nice catch I agree.

Reply:
But you know it would require additional conditions or code duplication because we use the same modal template for delete, sync collections, and students grade update operation and I suppose this change is not required now.